resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -4464,6 +4464,774 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
           <hashTree/>
         </hashTree>
       </hashTree>
+      <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Track Service Request (Preparation)" enabled="true">
+        <stringProp name="ThreadGroup.on_sample_error">continue</stringProp>
```

Comment:
Channel Verification Status (Preparation) - loop controller is hardcode to 2 times, what if the otp channels are more ?

Reply:
Now have defined a variable in user defined variable as "channelTypeLoopCount" where we can update the loop count with the number of channel types we are using.